book/src/pages/foreword-1.md

```diff
@@ -19,8 +19,11 @@ Since we're assuming you have little programming experience we won't go into the
 Suffice to say there are different ways to think about and write computer programs, and we've chosen the functional programming approach.
 
 The reasons for choosing functional programming are more interesting.
-It's common to teach programming by what we call the "bag of syntax" approach.
-In this approach a programming language is taught a collection of syntactical features (variables, for loops, while loops, methods) and students are left to figure out on their own when to use each feature.
+It's common to teach programming by what we call the "bag of syntax" approach, where "syntax" is the term we use for the rules defining the structure of symbols in a programming language. 
```

Comment:
Unfortunately, I feel this explanation is in the wrong place. It's in the foreword for the original version of Creative Scala which:

1. I don't think many people will read; and
2. I don't want to add new content to as I want to keep it as close to the original as makes sense.

(Updating links or making other corrections is fine.)

I think the first place that syntax is mentioned in the book proper is https://www.creativescala.org/creative-scala/expressions/04-types.html